dashboard/test/models/contact_rollups_pardot_memory_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
+require 'cdo/contact_rollups/v2/pardot'
+
+class ContactRollupsPardotMemoryTest < ActiveSupport::TestCase
+  test 'add_and_update_pardot_ids inserts new mappings' do
+    ContactRollupsPardotMemory.delete_all
```

Comment:
out of curiosity: is this the standard way to clean up for tests? does the transactional test case flag not clean things up?

Reply:
@hacodeorg wrote this, but my impression is that this isn't quite standard, but is pretty much foolproof :)

I asked last week in #test-tactics about whether to assume that tests are transactional, and it sounded like it's not 100%. So, something like this makes it 100%?

Reply:
This is to be on the safe side. From the discussion [here](https://codedotorg.slack.com/archives/CHZR21TLY/p1584383163006000), `use_transactional_test_case` is used in conjunction with `setup_all` and `teardown_all` hooks to rollback DB changes of the entire TestCase (a test file, not a test method). The code is [here](https://github.com/code-dot-org/code-dot-org/blob/23b9c75e79f6c9b94f3c6f9894030e7a57daacb5/dashboard/test/testing/transactional_test_case.rb).